src/soc/ibm/power9/i2c.c

```diff
@@ -0,0 +1,172 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <device/i2c_simple.h>
+#include <console/console.h>
+#include <cpu/power/scom.h>
+#include <spd_bin.h>
+
+#define FIFO_REG(bus)		(0xA0004 | ((bus) << 12))
+#define CMD_REG(bus)		(0xA0005 | ((bus) << 12))
+#define MODE_REG(bus)		(0xA0006 | ((bus) << 12))
+#define STATUS_REG(bus)		(0xA000B | ((bus) << 12))
```

Comment:
Please align the defines for better readability

Reply:
I'm not sure why GitHub displays it this way, this is properly aligned in the code. I changed the alignment of the following defines, there are now indented according to coreboot's coding style (8 chars wide).